config/environments/development.rb

```diff
@@ -29,7 +29,7 @@
   end
 
   # Store uploaded files on the local file system (see config/storage.yml for options).
-  config.active_storage.service = :local
+  config.active_storage.service = :cloudinary
```

Comment:
@georgkloeck - shouldn't we just use our local filesystem in development to store images?  In production, cloudinary, yes.  But in development, normally we would just use :local